test/runner.ml

```diff
@@ -6,4 +6,18 @@ let () =
     | [] | [_] -> None
     | _::l -> Some (List.map ExtString.String.lowercase l)
   in
+  let tests = [
+    Test_BitSet.register;
+    Test_Dllist.register;
+    Test_DynArray.register;
+    Test_ExtArray.register;
+    Test_ExtHashtbl.register;
+    Test_ExtList.register;
+    Test_ExtString.register;
+    Test_IO.register;
+    Test_std.register;
+  ]
+  in
+  List.iter (fun register -> register ()) tests;
```

Comment:
why is this needed? dune will not link if no explicit dep?

Reply:
I’ve just tested and no. To be more explicit (and true), I’ve updated the comment in

https://github.com/ygrek/ocaml-extlib/pull/64/files#diff-6c3a27ff8273715c892c921fcc5edb627d8bd48baae795a8b2f8a463ccd431a7R27

to show this fact